OpenTrons/opentrons-api
# fix(openrons-ai-server, opentrons-ai-client): predict method

# Overview
Two changed
- **Backend**: Previously, two endpoints end up using the same model, therefore confused during message processing. Present, two endpoints use separate models.
- **Frontend**: Flex gripper is concerned with only Flex robot 
   - opentrons-ai-client/src/resources/utils/createProtocolUtils.tsx 

Closes AUTH-1076


## Test Plan and Hands on Testing
- Visit `opentrons.ai` 
- Click `Update an existing protocol` and follow instructions
- Click `Create a new protocol` and follow instructions. Once you complete providing labware and other information. Click Submit then it will take you chat window where you need to see the generated protocol. It should not start like 'Simulation is successful'.


 


## Changelog



## Review requests
All tests are passing 


## Risk assessment
Low
